client/hosting/sites/components/site-preview-pane/preview-pane-header-buttons.tsx

```diff
@@ -14,24 +14,18 @@ type Props = {
 	sectionName?: string;
 };
 
-const PreviewPaneHeaderButtons = ( {
-	focusRef,
-	closeSitePreviewPane,
-	itemData,
-	sectionName,
-}: Props ) => {
+const PreviewPaneHeaderButtons = ( { focusRef, closeSitePreviewPane, itemData }: Props ) => {
 	const adminButtonRef = useRef< HTMLButtonElement | null >( null );
 	const { adminLabel, adminUrl } = useSiteAdminInterfaceData( itemData.blogId );
 	const { __ } = useI18n();
-	const isHostingOverview = sectionName === 'dotcom-hosting';
 
 	return (
 		<>
 			<Button onClick={ closeSitePreviewPane } className="item-preview__close-preview-button">
 				{ __( 'Close' ) }
 			</Button>
 			<Button
-				primary={ isHostingOverview }
+				primary
```

Comment:
This is what makes the button blue (the CTA). Everything else in this PR is reverting the supporting code added in https://github.com/Automattic/wp-calypso/pull/94192